terraform/52-packagefeedcreatereleasetrigger/project_triggers.tf

```diff
@@ -3,14 +3,18 @@ resource "octopusdeploy_external_feed_create_release_trigger" "external_feed_tri
   space_id    = var.octopus_space_id
   project_id  = "${octopusdeploy_project.deploy_frontend_project.id}"
   package {
-    deployment_action = octopusdeploy_deployment_process.example.step[0].run_script_action[0].name
+    deployment_action_slug = "hello-world-using-powershell-1"
```

Comment:
Any reason you changed this? I kind of prefer it the other way so you get the implicit depends on.

Reply:
Yes because slugs aren't defined in the `deployment_action` resource schema

Reply:
I'm also not sure that's something that should be in the resource schema either because slugs are meant to be auto-generated normally right?

Reply:
Oh dear, that's not good...

Slugs _can_ be autogenerated, but I'd want to the option to manually specify them. But I don't think that's something to solve in this PR.
 
Terraform can have a different data model to the model that you define it with - so at the very least I think we should add the slug to the deployment action that's returned, even if we don't expose setting it just yet.

Without that, the user needs to guess what the slug will be generated as and then leave it as a magic string.

Reply:
Yes that's true. I think those are 'computed values'?

Yea we should add them to the model in a future PR, I'll create a ticket for that